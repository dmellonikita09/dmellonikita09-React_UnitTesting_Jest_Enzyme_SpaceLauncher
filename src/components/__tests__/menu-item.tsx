import React from 'react';
import { mount } from 'enzyme';
import MenuItem from '../menu-item';

describe('Menu Item', () => {
  it('should render correctly', () => {
    const wrapper = mount(<MenuItem to="/wow" />);
    expect(wrapper).toMatchSnapshot();
  });
});