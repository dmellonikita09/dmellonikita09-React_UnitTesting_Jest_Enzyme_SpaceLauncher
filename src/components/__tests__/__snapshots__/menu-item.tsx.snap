// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Item should render correctly 1`] = `
<Styled(Link)
  to="/wow"
>
  <Link
    className="css-1yu82wf"
    to="/wow"
  >
    <Location>
      <LocationProvider
        history={
          Object {
            "_onTransitionComplete": [Function],
            "listen": [Function],
            "location": Object {
              "hash": "",
              "host": "localhost",
              "hostname": "localhost",
              "href": "http://localhost/",
              "key": "initial",
              "origin": "http://localhost",
              "pathname": "/",
              "port": "",
              "protocol": "http:",
              "search": "",
              "state": null,
            },
            "navigate": [Function],
            "transitioning": false,
          }
        }
      >
        <a
          className="css-1yu82wf"
          href="/wow"
          onClick={[Function]}
        />
      </LocationProvider>
    </Location>
  </Link>
</Styled(Link)>
`;
